content/en/guide/v10/getting-started.md

```diff
@@ -110,7 +110,7 @@ Upon completion, you'll have a new `dist/` folder which can be deployed directly
 
 ## Integrating Into An Existing Pipeline
 
-If you already have an existing tooling pipeline set up, it's very likely that this includes a bundler. The most popular choices are [webpack](https://webpack.js.org/), [rollup](https://rollupjs.org) or [parcel](https://parceljs.org/). Preact works out of the box with all of them. No changes needed!
+If you already have an existing tooling pipeline set up, it's very likely that this includes a bundler. The most popular choices are [webpack](https://webpack.js.org/), [rollup](https://rollupjs.org) or [parcel](https://parceljs.org/). Preact works out of the box with all of them, no major changes needed!
```

Comment:
Felt it was a bit silly to say "no changes needed" followed by how to make the necessarily changes.